build.sbt

```diff
@@ -1,20 +1,35 @@
+import ReleaseTransformations.*
+import sbtversionpolicy.withsbtrelease.ReleaseVersion
 import play.sbt.PlayImport.specs2
 import sbt.Keys.{libraryDependencies, mainClass}
 import sbtassembly.AssemblyPlugin.autoImport.{assemblyJarName, assemblyMergeStrategy}
 import sbtassembly.MergeStrategy
 import com.typesafe.sbt.packager.docker.{Cmd, ExecCmd}
 
 val projectVersion = "1.0-latest"
-
-organization := "com.gu"
+ThisBuild / publish / skip := true
```

Comment:
This repo has any projects, and almost all of them we _don't_ want to publish to Maven Central, so unusually for `[gha-scala-library-release-workflow](https://github.com/guardian/gha-scala-library-release-workflow)` we make `publish / skip := true` the _default_ - and then set it in `api-models` below.